app.js

```diff
@@ -0,0 +1,200 @@
+var preferences = {};
+var customerPreferences = {};
+var drinks = [];
+var customers = [];
+var currentCustomer;
+var isCustomer = false;
+var state = {
+    drinkNumber: 0,
+    yesCount: 0
+};
```

Comment:
I would declare these variables all together since they're empty like...

`var preferences = {},
customerPreferences = {},
drinks = [];
customers = [],
currentCustomer,
...`